sgconfig.yml

```diff
@@ -0,0 +1,4 @@
+ruleDirs:
+  - ./rules
+testConfigs:
+  - testDir: ./rule-tests
```

Comment:
Is this supposed to be included?  I think this is leftover from the AST Grep run